storage/file.go

```diff
@@ -1,3 +1,4 @@
+// Package storage provides storage specific functionality.
```

Comment:
I think this one could be better "storage specific" sounds like those per-platform comments above.
This however is features for all platforms.
How about "storage access and management functionality."